src/main/scala/com/fulcrumgenomics/umi/DuplexConsensusCaller.scala

```diff
@@ -256,4 +298,20 @@ class DuplexConsensusCaller(override val readNamePrefix: String,
 
     rec
   }
+
+
+  /**
+    * Logs statistics about how many reads were seen, and how many were filtered/discarded due
+    * to various filters.
+    */
+  override def logStatistics(logger: Logger): Unit = {
+    require(readsFilteredInsufficientSupport == 0, s"readsFilteredInsufficientSupport should be zero, was ${readsFilteredInsufficientSupport}%,d")
+    logger.info(f"Total Raw Reads Considered: ${totalReads}%,d.")
+    logger.info(f"Raw Reads Filtered Due to Being Fragment Reads: ${readsFilteredFragments}%,d.")
+    logger.info(f"Raw Reads Filtered Due to Observing AB/BA Reads Only: ${readsFilteredAb}%,d.")
+    logger.info(f"Raw Reads Filtered Due to Mismatching Alignments: ${readsFilteredMinorityAlignment}%,d.")
+    logger.info(f"Raw Reads Filtered Due to Creating an Orphan Consensus: ${readsFilteredSingleStrandConsensusOnly}%,d.")
+    logger.info(f"Fraction of Raw Reads in Consensus Filtered due to Mismatching Alignments: ${_readsPreCigarFilter / (_readsPreCigarFilter + _readsPostCigarFilter).toDouble}%.4f")
```

Comment:
I think this should be `(_readsPreCigarFilter - _readsPostCigarFilter) / (_readsPreCigarFilter + _readsPostCigarFilter)`